pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -411,45 +411,35 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				Reason: api.NodePodNumberExceeded,
 			}
 			predicateStatus = append(predicateStatus, podsNumStatus)
-			return predicateStatus, nil
```

Comment:
If we don't return, please filter out those reason is null string at 
https://github.com/volcano-sh/volcano/blob/c91eb07f2c300e4d5c826ff11a63b91781b3ac11/pkg/scheduler/util/predicate_helper.go#L156-L158

Reply:
done